users/urls.py

```diff
@@ -0,0 +1,7 @@
+from django.urls import path
+from . import views
+
+urlpatterns = [
+    path('profile/', views.profile, name='Users-profile'),
+    path('login/', views.login, name='Users-login'),
```

Comment:
Although path names aren't [restricted to valid Python names](https://docs.djangoproject.com/en/4.0/topics/http/urls/#naming-url-patterns-1), I still think it should be lowercase